### ISSUE TITLE
Server crashes just by opening a Swift file

### ISSUE THREAD

**Issue by alexito4** [owensd/vscode-swift#3]
I just installed the extension and created a new Swift project with SPM. And just opening the main swift file makes VSC show a message at the top:

> The Swift Language Server server crashed 5 times in the last 3 minutes. The server will not be restarted.

Checking the output of the Swift Language Server shows this:

```
dyld: Library not loaded: @rpath/sourcekitd.framework/Versions/A/sourcekitd
  Referenced from: /Users/alejandromartinez/.vscode/extensions/kiadstudios.apous-0.5.2/lib/usr/bin/langsrv
  Reason: image not found
[Info  - 3:49:08 PM] Connection to server got closed. Server will restart.
dyld: Library not loaded: @rpath/sourcekitd.framework/Versions/A/sourcekitd
  Referenced from: /Users/alejandromartinez/.vscode/extensions/kiadstudios.apous-0.5.2/lib/usr/bin/langsrv
  Reason: image not found
[Info  - 3:49:08 PM] Connection to server got closed. Server will restart.
dyld: Library not loaded: @rpath/sourcekitd.framework/Versions/A/sourcekitd
  Referenced from: /Users/alejandromartinez/.vscode/extensions/kiadstudios.apous-0.5.2/lib/usr/bin/langsrv
  Reason: image not found
[Info  - 3:49:08 PM] Connection to server got closed. Server will restart.
dyld: Library not loaded: @rpath/sourcekitd.framework/Versions/A/sourcekitd
  Referenced from: /Users/alejandromartinez/.vscode/extensions/kiadstudios.apous-0.5.2/lib/usr/bin/langsrv
  Reason: image not found
[Info  - 3:49:08 PM] Connection to server got closed. Server will restart.
dyld: Library not loaded: @rpath/sourcekitd.framework/Versions/A/sourcekitd
  Referenced from: /Users/alejandromartinez/.vscode/extensions/kiadstudios.apous-0.5.2/lib/usr/bin/langsrv
  Reason: image not found
[Error - 3:49:08 PM] Connection to server got closed. Server will not be restarted.
```

**Comment by owensd** [comment]
Ive seen some reports of this. What version and how did you install Swift?

The problem is that it can't find sourcekit.

**Comment by alexito4** [comment]
The one that comes with Xcode 8.3.2, installed from the App Store.
I also have a couple of snapshots installed (via the mac installed) in case it may affect.

<img width="674" alt="screen shot 2017-06-03 at 16 49 07" src="https://cloud.githubusercontent.com/assets/750912/26754986/93f5e7cc-487c-11e7-9247-e7c5145ec4e0.png">

**Comment by owensd** [comment]
Yeah, sorry. I've been testing with the official toolchain versions instead. Downloading Xcode 8.3.2 now make sure it has everything I need.

**Comment by owensd** [comment]
Ok, I got it installed and verified that this is working for me. You can configure your toolchain path, as the README shows. You'll need to point that Xcode's location until I add logic to do lookups in multiple places.

So in your settings.json file for VS Code, add the following:

`"swift.toolchainPath": "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain"`

Reload the window, and that should fix it. Sorry about the troubles.

**Comment by owensd** [comment]
This should be fixed in v0.5.3 now. No need for the setting changes.
